opba-db/src/test/java/de/adorsys/opba/db/BankProtocolActionsSqlGeneratorTest.java

```diff
@@ -0,0 +1,149 @@
+package de.adorsys.opba.db;
+
+import com.google.common.io.FileWriteMode;
+import com.google.common.io.Files;
+import com.google.common.io.Resources;
+import de.adorsys.opba.db.config.TestConfig;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.List;
+import java.util.UUID;
+
+import static de.adorsys.opba.db.BankProtocolActionsSqlGeneratorTest.ENABLE_BANK_PROTOCOL_ACTIONS_SQL_GENERATION;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Is not actually a test. This class generates 'banks_random_data.csv' out of 'banks.csv',
```

Comment:
Please correct JavaDoc - it does not generate `banks_random_data`

Reply:
JavaDoc was fixed